ptsprojects/stack.py

```diff
@@ -317,6 +317,47 @@ def __init__(self, uuid, oob, output_size, output_actions, input_size,
         # Node Identity
         self.proxy_identity = False
 
+        # MMDL status data
+        self.status_data = Property({
+            "Ack": True,
+            'Status': [],
+        })
+
+        self.rcv_status_data = Property({
+            "Ack": True,
+            'Status': [],
+        })
+
+    def rcv_status_data_set(self, key, data):
+        if key in self.rcv_status_data.data:
```

Comment:
I'm not sure I understand how it should work :smile:
This function seems to "update" the value in dict. If the key is not in dict it logs an error only.
Where is the place you actually add a key to dict?
The same applies to status_data_set

Reply:
```
Where is the place you actually add a key to dict?
```
It is defined a few lines above. `Ack` and `Status` are the only things in the dict.